features/features_test.go

```diff
@@ -216,3 +217,66 @@ func (s *featureSuite) TestFlag(c *C) {
 	_, err = features.Flag(tr, features.Layouts)
 	c.Assert(err, ErrorMatches, `layouts can only be set to 'true' or 'false', got "banana"`)
 }
+
+func (s *featureSuite) TestAll(c *C) {
```

Comment:
I won't block the PR for it as it is tested but it would be better if this test didn't depend on the inner workings of specific flags like QuotaGroups since it might be broken by changing/adding callbacks. We could export `featuresSupportedCallbacks` through `export_test.go` and then set custom callbacks to test whatever we want (enabled but unsupported, support but disabled, etc) and check `All`'s output

Reply:
Thanks, and good idea, I'll add a task for myself to improve these tests in the future.